src/site/includes/appeals-banner.html

```diff
@@ -0,0 +1,15 @@
+  <div class="usa-alert-full-width usa-alert-full-width-warning">
+    <div aria-live="polite" role="alert" class="usa-alert usa-alert-warning">
+      <div class="usa-alert-body">
+        <h3 class="usa-alert-heading">The Appeals process has changed</h3>
+        <div class="usa-alert-text">
+          <b>For VA decisions received aftger February 19, 2019</b>, please keep reading below to learn about your decision review options.
+          <br>
+          <br>
+          <b>For VA decisions received before February 19, 2019</b>, you'll need to follow a different process to appeal the decision.
+          <br>
+          <a href="#">Learn how to file an appeal</a>.
```

Comment:
Is this being controlled with JavaScript, or should the `href` have a path?

Reply:
# was just a placeholder. I updated the `<a href>` tag to go link to `/decision-reviews/`